components/table/table-wrapper.js

```diff
@@ -258,10 +264,14 @@ export class TableWrapper extends RtlMixin(PageableMixin(SelectionMixin(LitEleme
 				--d2l-table-border-radius: 0rem; /* stylelint-disable-line length-zero-no-unit */
 				--d2l-table-border-radius-sticky-offset: 0rem; /* stylelint-disable-line length-zero-no-unit */
 				--d2l-table-cell-height: 1.15rem; /* min-height to be 48px including border */
-				--d2l-table-cell-padding: 0.6rem;
-				--d2l-table-cell-padding-alt: calc(0.6rem - 1px) 0.6rem 0.6rem 0.6rem;
+				--d2l-table-cell-padding: 0.75rem;
+				--d2l-table-cell-padding-alt: calc(0.75rem - 1px) 0.75rem 0.75rem 0.75rem;
```

Comment:
Did you confirm all this with Nicole? This modifies the padding on all sides for light style for the regular table cells, whereas for default you only modified the left/right.

Reply:
The spec is specifically for the sort header though, so it would be good to confirm if it's also all cells.
If that is the case then the default padding would also need updating.

Reply:
I sent her another message about this specifically but this is what she told me before:

Yes, the adjustment to the cell padding was intended across all header cells - and I suppose that might make sense to apply across all cells so that the header content lines up with the cell content below it